README.md

```diff
@@ -192,9 +192,10 @@ html_body = <<-HTML.strip
 </table>
 HTML
 context = {
-  article: Sablon.content(:html, html_body) }
-  # alternative method using special key format
-  # 'html:article' => html_body
+  article: Sablon.content(:html, html_body) },
+  # Or use html: prefix to make sablon parse the value as HTML.
+  # Does the same as above.
+  'html:article' => html_body
```

Comment:
@siegy22 i would consider to go even further and add a separate example with a short intro description. i think that would make it more visible.

Reply:
I'm fine with what is currently shown, or a more explicit example as mentioned by @deiferni. I'll leave it up to you @siegy22.